DUOS-1047[risk=no] Added styling to New Chair Console Buttons

JVThomas opened the pull request:
Addresses [DUOS-1047] (https://broadworkbench.atlassian.net/browse/DUOS-1047)

PR adds styling to the election buttons via Theme style updates and the introduction of two components: one for table buttons depicted with text labels and another for table buttons depicted with svg icons. Code also corrects miscount on total votes for Open elections by filtering for the current election id.

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] PR is labeled with a Jira ticket number and includes a link to the ticket
- [x] PR is labeled with a security risk modifier [no, low, medium, high] 
- [x] PR describes scope of changes

In all cases:

- [x] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [x] Get PO sign-off for all non-trivial UI or workflow changes
- [x] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment


[DUOS-1047]: https://broadworkbench.atlassian.net/browse/DUOS-1047

JVThomas commented on src/libs/utils.js:
```diff
@@ -3,6 +3,13 @@ import 'noty/lib/noty.css';
+
+export const applyHoverEffects = (e, style) => {
+  forEach(style, (value, key) => {
+    e.target.style[key] = value;
```
Codacy is flagging this line of code as a possible injection vector. I would like to state that this is a false positive since ```key``` is defined within the code and is never provided by the user. Is it okay to waive this issue in Codacy?

rushtong reviewed (approved):
Changes look amazing 👍🏽

rushtong commented:
Need to make sure automation checks pass before merging.

rushtong commented:
@JVThomas 
I re-ran codacy and that's passing now. The build, however, is failing:
```
<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x432f0cdbe1d]
    1: StubFrame [pc: 0x432f0c934b0]
Security context: 0x24e34a69e6e9 <JSObject>
    2: /* anonymous */(aka /* anonymous */) [0x12412e041399] [/root/project/node_modules/webpack-sources/lib/applySourceMap.js:~58] [pc=0x432f25aa7de](this=0x1a45406826f1 <undefined>,chunk=0x38145da9b6b1 <String[3]: 24|>,middleMapping=0x108309f8bd01 <Object map = 0x3aa117c8d469>)
    3: SourceNode_walk [0x1987ba1a3421] [/r...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0x8fa0c0 node::Abort() [node]
 2: 0x8fa10c  [node]
 3: 0xb0026e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0xb004a4 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0xef49b2  [node]
 6: 0xef4ab8 v8::internal::Heap::CheckIneffectiveMarkCompact(unsigned long, double) [node]
 7: 0xf00b92 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [node]
 8: 0xf014c4 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
 9: 0xf04131 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [node]
10: 0xecd5b4 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationSpace) [node]
11: 0x116d73e v8::internal::Runtime_AllocateInNewSpace(int, v8::internal::Object**, v8::internal::Isolate*) [node]
12: 0x432f0cdbe1d 

Exited with code exit status 1
CircleCI received exit code 1
```
I'm not sure how to solve this problem, but will try to devote some debugging time to it.

JVThomas commented:
Here's an issue that details the circle-ci issue:
https://github.com/webpack/webpack-sources/issues/66

And the solution in the thread:
https://github.com/webpack/webpack-sources/issues/66#issuecomment-616880715

rushtong commented:
@JVThomas PR is now passing in Circleci with the increased memory setting. 👍🏽

JVThomas commented:
NOTE: npm audit flag on immer (dependency of react-scripts) is a false positive

JVThomas commented:
Sorry, added some additional changes to styling. I added a word-break attribute to the title cell styling for datasets that are essentially one word long but have a lot of characters. Also added some minor styling adjustments to election buttons to make it look less cramped in the row. Would appreciate a re-review on the new changes. You can see the word-break issue on staging (ton of titles that use underscores instead of spaces, so it ends up being a one word title with 80+ characters).

rushtong reviewed (approved):
Still 👍🏽
